apps/calendar/src/factory/calendarCore.tsx

```diff
@@ -33,14 +33,142 @@ import type {
+ * @typedef {CustomEvents} CustomEvents
+ */
+
+/**
+ * Define Calendars to group events.
+ *
+ * @typedef {object} CalendarInfo
+ * @property {string} id - Calendar id.
+ * @property {string} name - Calendar name.
+ * @property {string} color - Text color of events.
+ * @property {string} borderColor - Left border color of events.
+ * @property {string} backgroundColor - Background color of events.
+ * @property {string} dragBackgroundColor - Background color of events during dragging.
+ */
+
+/**
+ * Timezone options of the calendar instance.
+ *
+ * For more information, see {@link https://github.com/nhn/tui.calendar/blob/main/docs/ko/apis/options.md#timezone|Timezone options} in guide.
+ *
+ * @typedef {object} TimezoneOptions
+ * @example
+ * const calendar = new Calendar('#container', {
+ *   timezone: {
+ *     // @property {string} zones[].timezoneName - Timezone name. it should be one of IANA timezone names.
+ *     // @property {string} [zones[].displayLabel] - Display label of timezone.
+ *     // @property {string} [zones[].tooltip] - Tooltip of the element of the display label.
+ *     zones: [
+ *       {
+ *         timezoneName: 'Asia/Seoul',
+ *         displayLabel: 'UTC+9:00',
+ *         tooltip: 'Seoul'
+ *       },
+ *       {
+ *         timezoneName: 'Europe/London',
+ *         displayLabel: 'UTC+1:00',
+ *         tooltip: 'BST'
+ *       }
+ *     ],
+ *     // This function will be called for rendering components for each timezone.
+ *     // You don't have to use it if you're able to `Intl.DateTimeFormat` API with `timeZone` option.
+ *     // this function should return timezone offset from UTC.
+ *     // for instance, using moment-timezone:
+ *     customOffsetCalculator: (timezoneName, timestamp) => {
+ *       return moment.tz(timezoneName).utcOffset(timestamp);
+ *     }
+ *   }
+ * });
+ * @property {Array.<object>} zones - Timezone data.
+ * @property {string} zones[].timezoneName - Timezone name. it should be one of IANA timezone names.
+ * @property {string} [zones[].displayLabel] - Display label of timezone.
+ * @property {string} [zones[].tooltip] - Tooltip of the element of the display label.
+ * @property {function} customOffsetCalculator - Custom offset calculator when you're not able to leverage `Intl.DateTimeFormat` API.
+ */
+
+/**
+ * Object to create/modify events.
+ * @typedef {object} EventObject
+ * @property {string} [id] - Event id.
+ * @property {string} [calendarId] - Calendar id.
+ * @property {string} [title] - Event title.
+ * @property {string} [body] - Body content of the event.
+ * @property {string} [isAllday] - Whether the event is all day or not.
+ * @property {string|number|Date|TZDate} [start] - Start time of the event.
+ * @property {string|number|Date|TZDate} [end] - End time of the event.
+ * @property {number} [goingDuration] - Travel time which is taken to go.
+ * @property {number} [comingDuration] - Travel time which is taken to come back.
```

Comment:
```suggestion
 * @property {number} [goingDuration] - Travel time which is taken to go in minutes.
 * @property {number} [comingDuration] - Travel time which is taken to come back in minutes.
```

분 단위의 숫자임을 알려주면 좋을 것 같아요.